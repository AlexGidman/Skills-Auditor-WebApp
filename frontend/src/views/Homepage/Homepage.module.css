.header {
    margin-top: 10%;
}

.welcomeMessage {
    font-size: xx-large;
}
