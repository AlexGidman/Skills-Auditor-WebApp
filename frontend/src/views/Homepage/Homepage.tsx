import React from "react";

import styles from "./Homepage.module.css";

export const Homepage = () => {
    return (
        <>
            <header className={styles.header}>
                <h1>Welcome to Skills Auditor!</h1>
            </header>
            <section className={styles.welcomeMessage}>
                Click on navigation options to view details, add skills, and optimise your team
            </section>
        </>
    );
};
