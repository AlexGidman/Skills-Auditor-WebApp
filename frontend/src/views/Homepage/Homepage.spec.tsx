import { render, screen } from "@testing-library/react";
import React from "react";
import { Homepage } from "./Homepage";

describe("Homepage", () => {
    it("renders correctly", async () => {
        const { container } = render(<Homepage />);
        expect(
            await screen.findByRole("heading", { name: "Welcome to Skills Auditor!" }),
        ).toBeInTheDocument();
        expect(container).toMatchSnapshot();
    });
});
