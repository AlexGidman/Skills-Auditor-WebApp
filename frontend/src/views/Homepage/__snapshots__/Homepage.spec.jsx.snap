// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Homepage renders correctly 1`] = `
<div>
  <header
    class="header"
  >
    <h1>
      Welcome to Skills Auditor!
    </h1>
  </header>
  <section
    class="welcomeMessage"
  >
    Click on navigation options to view details, add skills, and optimise your team
  </section>
</div>
`;
